vars/main.yml

```
@@ -5,6 +5,7 @@ core:
   ran_subnet: "172.20.0.0/16" # set it to empty to get subnet from 'data_iface'
 
   helm:
+    local_charts: false
     chart_ref: aether/sd-core
     chart_version: 0.12.6
```

Update version here:
```suggestion
    chart_version: 1.0.4
```